c/meterpreter/source/extensions/stdapi/server/sys/process/process.c

```diff
@@ -860,6 +860,91 @@ DWORD request_sys_process_kill(Remote *remote, Packet *packet)
 	return ERROR_SUCCESS;
 }
 
+DWORD request_sys_process_nkill(Remote *remote, Packet *packet)
+{
+	Packet *response = packet_create_response(packet);
+	DWORD result = ERROR_SUCCESS;

```

Comment:
Given that this isn't supported in POSIX, a better default should be used here that indicates something isn't supported.